web/src/main/webapp/resources/javascript/cart.js

```diff
@@ -38,4 +38,34 @@ function addToCart(phoneId, quantity) {
     });
     message.append(messageHead, messageBody);
     document.querySelector('#statusMessage').replaceChildren(message);
+}
+
+function deleteFromCart(phoneId) {
+    const message = document.createElement('div');
```

Comment:
Id' better recommend you to change a bit this logic. You can create all this content on pages itself with display:none and some css class like success/error. After this you will need only to change display property and select proper styling